checker/src/main/java/org/checkerframework/checker/calledmethods/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,50 @@
+package org.checkerframework.checker.calledmethods.framework;
+
+import com.sun.source.tree.NewClassTree;
+import javax.lang.model.element.ExecutableElement;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+/** add CalledMethods annotations to code generated by framework */
+public interface FrameworkSupport {
```

Comment:
The term "framework" is too broad.  It's confusing in a project named the "Checker Framework" that has a component called "framework".  I suggest that you change "framework" to "builder framework" (and Framework to BuilderFramework, etc.) throughout.  I think that will be clearer.